### DISCUSSION TITLE
Square AppLogo

### DISCUSSION THREAD

**Discussion by BlafKing** [Windos/BurntToast#167] [Q&A]
Hi all!
I've been trying to use a square Icon as the app logo for a notification.
This is the notification script im currently using:
```
$Message = Read-Host -Prompt 'Message here'
$Image = New-BTImage -Source 'F:\tombo\Desktop\wozebe.png' -Crop 'None'
$Noti = @{
    Text    =   'Test', "$Message"
    AppLogo =   "$Image"
}
New-BurntToastNotification @Noti
```
The Icon always shows up as a circle, I've tried using -crop 'Default' & -crop 'None'
the image I'm using has a size of 256x256
Any help would be much appreciated, thanks in advance! :)

**Comment by Windos** [comment]
I'm actually kind of impressed that worked at all (and I really should have done a better documenting and giving guidance on this).

The `-AppLogo` parameter on `New-BurntToastNotification` is only meant to accept the path to an image, not a "BTImage" object in and of itself. Behind the scenes that path is used to create a new BTImage with a hard coded circle crop.

If you're interested the code for that is here: https://github.com/Windos/BurntToast/blob/0e647199d78f5d1353eb1c9d276121007be1fab4/BurntToast/Public/New-BurntToastNotification.ps1#L201

I imagine the code is just taking the path out of the BTImage passed in and creating a new one.

Unfortunately, there's no easy way around that atm. You'll need to use the "advanced functions" route.

You can see an example of this here: https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/01_Starter.ps1

I'm currently working on a v1 of BurntToast which is going to dramatically change (and simplify) things... this is one of the things that are about to be a heck of a lot easier.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
